.github/workflows/contracts-ecdsa.yml

```diff
@@ -252,9 +252,24 @@ jobs:
 
       - name: Deploy contracts
         env:
-          CHAIN_API_URL: ${{ secrets.GOERLI_ETH_HOSTNAME_HTTP }}
-          ACCOUNTS_PRIVATE_KEYS: ${{ secrets.GOERLI_ETH_CONTRACT_OWNER_PRIVATE_KEY }}
-          ETHERSCAN_API_KEY: ${{ secrets.ETHERSCAN_API_KEY }}
+          # Use fake ternary expressions to decide which credentials to use,
+          # depending on chosen environment. Note: if `GOERLI...` credentials
+          # are empty, the expressions will be evaluated to the `SEPOLIA...`
+          # ones.
```

Comment:
Same question as in https://github.com/threshold-network/solidity-contracts/pull/151#discussion_r1324225132

Reply:
Addressed (074739e858aa1eb7a2897097bbe4cc9b7ac51da4 and 63d086d5394e83ba014bcddfbd083580de4183a8)